tests/test_bug_features.py

```diff
@@ -163,6 +165,19 @@ def test_comment_length(read):
     read("comment_length.json", comment_length, [566, 5291])
 
 
+def test_bug_extractor():
+    feature_extractors = [blocked_bugs_number(), bug_reporter(), bug_reporter()]
+    cleanup_functions = [
+        fileref(),
+        url(),
+        url(),
+    ]
+    try:
```

Comment:
Your code is not making sure that the assertion is thrown. You'd have to do something like:
```
try:
    BugExtractor(feature_extractors, cleanup_functions)
    assert False, "An assertion should have failed"
except AssertionError:
    pass
```

Even better, use the pytest utility to do this, like in:
https://github.com/mozilla/bugbug/blob/abf47d3b1c69e782450a57918d5950b2bec22cb4/tests/test_db.py#L82-L83